packs/equipment-effects/effect-witchs-finger.json

```
@@ -0,0 +1,42 @@
+{
+    "_id": "sHOx852VoPinsQGY",
+    "img": "icons/commodities/gems/gem-rough-tapered-red.webp",
+    "name": "Effect: Witch's Finger",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Witch's Finger]</p>\n<p>For 1 hour, you have cold resistance 3.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Witch's Finger]</p>\n<p>You gain cold resistance 3.</p>"
```
No need to list the duration since that's in the sidebar as structured data